etc/module.xml

```diff
@@ -10,6 +10,6 @@
 -->
 
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-  <module name="SDM_Altapay" setup_version="3.2.1">
+  <module name="SDM_Altapay" setup_version="3.2.2">
```

Comment:
Missing release notes

Reply:
I have already updated the release notes which is recently merged.